src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/AgencyInternalLocation.java

```diff
@@ -103,6 +103,7 @@ public class AgencyInternalLocation {
     private Integer capacity;
 
     @OneToMany(fetch = FetchType.LAZY)
+    @Where(clause = "INT_LOC_PROFILE_TYPE = 'HOU_UNIT_ATT' AND INT_LOC_PROFILE_CODE is not NULL")
     @JoinColumn(name = "INTERNAL_LOCATION_ID", referencedColumnName = "INTERNAL_LOCATION_ID")
     private List<AgencyInternalLocationProfile> profiles;
```

Comment:
so is there some associated Java code that was *always* applying this exact filter in all scenarios ?

Reply:
Yep, filtered on:
 ```
public boolean isAttribute() {
        return profileType.equals(HousingAttributeReferenceCode.DOMAIN) && housingAttributeReferenceCode != null;
}
```